src/interfaces/sdkTypes.ts

```diff
@@ -55,4 +55,105 @@ export interface LeaderboardUserData {
   totalVolumeInUsdShorts: Decimal;
   totalRealizedPnlPositions: Decimal;
   totalAccruedBorrowingFeesInUsd: Decimal;
-}
\ No newline at end of file
+} 
+export type Market = {
+  id: string;
+  marketName: string;
+  marketSymbol: string;
+  size: string;
```

Comment:
Add fields for formatted size. Please check with @iamamitkumar0512 to add other fields that we need to calculate to determine the skew and OI of markets for longs and shorts